user-service/src/main/java/com/movie/recommender/user/config/SecurityConfig.java

```diff
@@ -21,7 +21,7 @@ public SecurityFilterChain securityFilterChain(HttpSecurity http) throws Excepti
                 .sessionManagement(session -> session
                         .sessionCreationPolicy(SessionCreationPolicy.STATELESS))
                 .authorizeHttpRequests(authorize -> authorize
-                        .requestMatchers("/api/v1/users/register", "/api/v1/users/login").permitAll()
+                        .requestMatchers("/api/v1/users/register", "/api/v1/users/login", "/api/v1/users/{userId}/favorites").permitAll()
```

Comment:
For now I added here endpoints for test, because i dont have token filter. Option is to make all endpoints public